pkg/common/file.go

```diff
@@ -55,7 +55,7 @@ func CopyFile(src, dst string) (err error) {
 	if err != nil {
 		return fmt.Errorf("error reading source file: %w", err)
 	}
-	defer in.Close()
+	defer in.Close() // #nosec G307
```

Comment:
🙄 IMO disable this check entirely. (Minor nit, not blocking this PR.)

Reply:
Will do in another PR and discuss there.